src/main/java/org/zalando/failsafeactuator/aspect/Failsafe.java

```diff
@@ -1,29 +1,42 @@
 /**
- * The MIT License (MIT)
- * Copyright (c) 2016 Zalando SE
+ * The MIT License (MIT) Copyright (c) 2016 Zalando SE
```

Comment:
I don't believe that you actually need to put the header in every file.

Reply:
True, see #52